app/views/likes/create.js.erb

```diff
@@ -0,0 +1,3 @@
+$("#likes_<%= @recipe.id %>").html('<%= @recipe.likes.count %>&nbsp;');
```

Comment:
I see, you're using the @recipe var in here, so you can resolve the previous comments asking where it is used

Reply:
done!